src/modules/navigation/duck/reducer.js

```diff
@@ -24,4 +41,8 @@ export default combineReducers({
  * @param {Object} state - The state object containing the view property.
  * @returns {string} The sort value.
  */
-export const getSort = ({ view }) => view.sort
+// Selector needs to point to the correct state slice (`view`)
+export const getSort = state => state.view.sort
+
+// Selector for the new state (`view`)
```

Comment:
We should probably change this comment, the new state is new now, but won't be tomorrow